server/pages/index.js

```diff
@@ -528,6 +528,60 @@ module.exports = function() {
 		} );
 	} );
 
+	app.get( '/support-user', function( req, res ) {
+		// Do not iframe
+		res.set( {
+			'X-Frame-Options': 'DENY',
+		} );
+
+		// TODO: replace all render404s here with new template
+
+		if ( ! req.cookies.wordpress_logged_in ) {
+			// Not logged in, 404!
```

Comment:
can probably update this comment to ignore passed in tokens

Reply:
We can probably tidy some of the other comments/remove them too

Reply:
Updated in latest version.